src/components/Home/Home.js

```diff
@@ -116,16 +127,50 @@ const Home = () => {
       <LoadingSpinner loading={loading} />
 
       {!loading && (
-        <div>
-          <PhotoFeatureSection
-            subhead="Featured Collections"
-            headline="Collections"
-            linkTo="/collections"
-            linkToText="View All Collections"
-            items={galleryCollections}
-            data-testid="section-featured-collections"
-          />
-        </div>
+        <>
+          <div>
+            <PhotoFeatureSection
+              subhead="Featured Collections"
+              headline="Collections"
+              linkTo="/collections"
+              linkToText="View All Collections"
+              items={galleryCollections}
+              data-testid="section-featured-collections"
+            />
+          </div>
+          <section className="section">
```

Comment:
Looks great.  If we like this feature then we can abstract it to it's own component.

Reply:
I agree.